cypress/fixtures/allowable-actions/ben-active.json

```diff
@@ -0,0 +1,64 @@
+{
+  "couldFile": {
+    "filing": {
+      "filingTypes": [
+        {
+          "displayName": "Request for AGM Extension",
+          "name": "agmExtension"
+        },
+        {
+          "displayName": "AGM Location Change",
+          "name": "agmLocationChange"
+        },
+        {
+          "displayName": "Alteration",
+          "name": "alteration"
+        },
+        {
+          "displayName": "Amalgamation Application (Regular)",
+          "name": "amalgamationApplication",
+          "type": "regular"
+        },
+        {
+          "displayName": "Amalgamation Application Short-form (Vertical)",
+          "name": "amalgamationApplication",
+          "type": "vertical"
+        },
+        {
+          "displayName": "Amalgamation Application Short-form (Horizontal)",
+          "name": "amalgamationApplication",
+          "type": "horizontal"
+        },
+        {
+          "displayName": "Annual Report",
+          "name": "annualReport"
+        },
+        {
+          "displayName": "Address Change",
+          "name": "changeOfAddress"
+        },
+        {
+          "displayName": "Director Change",
+          "name": "changeOfDirectors"
+        },
+        {
+          "displayName": "6-Month Consent to Continue Out",
+          "name": "consentContinuationOut"
```

Comment:
There is also Consent for Amalgamation Out filing and and Amalgamation Out filing that are implemented in Business Filings UI but not released. Please create a new ticket to implement those if not already done.

Reply:
In UI If you see file: `src/components/bcros/filing/addStaffFiling/Index.vue` line 117 we supported this button even before. Additionally, from this PR we will be using API to verify if the button should be displayed or not.

Reply:
I see it! Thanks for the response. Resolving.